src/core_ocean/shared/mpas_ocn_vmix.F

```diff
@@ -1078,6 +1221,13 @@ subroutine ocn_vmix_implicit(dt, meshPool, diagnosticsPool, statePool, forcingPo
             dt, kineticEnergyCell, &
             vertViscTopOfEdge, layerThickness, layerThicknessEdge, normalVelocity, &
             ssh, bottomDepth, err)
+        else if (config_use_implicit_bottom_roughness) then
+          ! update bottomDrag via Cd=g*n^2*h^(-1/3)
```

Comment:
Is this formula used?   This comment seems like it was left over from a copy / paste but I may be missing something here.